azure-pipelines.yml

```diff
@@ -99,6 +99,26 @@ stages:
         projects: |
           $(Build.SourcesDirectory)/src/MIDebugEngine-Unix.sln
         arguments: '-c Debug'
+    - script: |
+        sudo apt-get install gdb g++ -y
+        which g++
+        which gdb
+        echo 0 | sudo tee /proc/sys/kernel/yama/ptrace_scope
+        # TODO: Try to make core dumps work
+        # echo 1 | sudo tee /proc/sys/kernel/core_uses_pid
+        # ulimit -S -c unlimited
+        # sudo sysctl -w kernel.core_pattern=$(Build.SourcesDirectory)/core.%e
+    - script: |
+        dotnet publish $(Build.SourcesDirectory)/src/OpenDebugAD7/OpenDebugAD7.csproj -c Debug -r linux-x64 --self-contained
```

Comment:
Lines 112-116 will be converted to an `sh` script in a future PR that includes publishing.